f2/src/locales/en.json

```diff
@@ -17,64 +17,53 @@
   "button.cancelReport": "Cancel report",
   "button.edit": "Edit",
   "confirmationPage.ImpactTitle": "What could be affected?",
-  "confirmationPage.ImpactTitle.edit":"Edit What Was Affected",
```

Comment:
I'm not sure how these lines got deleted - did you mean to?

Reply:
yeah, this is messing up our aria-labels
<img width="302" alt="image" src="https://user-images.githubusercontent.com/8228248/73011692-c24dc000-3de2-11ea-96c9-3912e194bb76.png">

oh! I bet it's because we didn't mark the text in the aria-labels for lingui to see when we made them bilingual.